tardis/resources/dronestates.py

```diff
@@ -226,20 +226,38 @@ class ShuttingDownState(State):
 
 
 class CleanupState(State):
+    transition = {ResourceStatus.Stopped: lambda: CleanupState(),
+                  ResourceStatus.Deleted: lambda: DownState(),
+                  ResourceStatus.Error: lambda: CleanupState()}
```

Comment:
Please consider simplifying the transition, i.e.

```suggestion
                  ResourceStatus.Error: CleanupState}
```

Reply:
There are use-cases where the lambda function is needed and I would like to keep it unique. For example: https://github.com/MatterMiners/tardis/blob/63e186576c658c0e3c9f4fe38e065aeebb16228b/tardis/resources/dronestates.py#L94-L105

Nevertheless, I am not yet happy how the state transitions are currently implemented. Issue has been opened #99.